<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinholes6</class>
 <widget class="QWidget" name="pinholes6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Pinholes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Horizontal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;(PHHORES6L)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L:stMotor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.STOP</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>IN</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.TWR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>OUT</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.TWF</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHVERES6L:stMotor</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>OUT</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHVERES6L.TWR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertikal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>IN</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHVERES6L.TWF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.STOP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Position and        Execute</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:cmdMode</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:rclMode</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>100u + Diode</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:cmdMode</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>50u</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:cmdMode</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>100u</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:cmdMode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>refernce all:</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref.</string>
   </property>
   <property name="pv" stdset="0">
    <string>APERTCS6G:cmdHome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>DIODEFS6L</string>
   </property>
  </widget>
  <widget class="QCALCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="pv" stdset="0">
    <string>DIODEFS6L:rdCur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>horiz. scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>560</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>vert. scan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.STOP</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="pin1Axis" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>345</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>PHHORES6L.VAL</string>
   </property>
   <property name="units" stdset="0">
    <string>mm</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.RBV</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="pin2Axis" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>345</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>PHVERES6L.VAL</string>
   </property>
   <property name="units" stdset="0">
    <string>mm</string>
   </property>
   <property name="pv" stdset="0">
    <string>PHVERES6L.RBV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;(PHVERSES6L)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="pin1Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>345</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>PHHORES6L.TWV</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>PHHORES6L.TWF</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="pin2Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>345</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>PHVERES6L.TWV</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>PHVERES6L.TWF</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCALCDNumber</class>
   <extends>QLCDNumber</extends>
   <header>qcalcdnumber.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
